src/borg/helpers.py

```diff
@@ -1239,14 +1253,23 @@ def progress(self, current=None, increase=1):
             self.trigger_at += self.step
             return pct
 
-    def show(self, current=None, increase=1):
+    def show(self, current=None, increase=1, info=[]):
```

Comment:
please use info=None as default and check for "is not None" below.

if you use a mutable datatype (like []) as default, checkers will complain as this can be problematic.